include/wx/aui/framemanager.h

```diff
@@ -654,7 +654,8 @@ class WXDLLIMPEXP_AUI wxAuiManager : public wxEvtHandler
     bool m_updateOnRestore = false;
 
 #ifndef SWIG
-    wxDECLARE_EVENT_TABLE();
+    void FrameBind();
+    void FrameUnbind();
```

Comment:
Do we really need these functions? We seem to be calling them from a single place only...

If we do, I think they should be called something like `{Bind,Unbind}FrameEventHandlers()` as "frame bind" reads awkwardly to me.

Reply:
They are indeed only called from one place each, but I created the FrameBind() function to closely parallel the original event table to make clear how the old and new code were related.  However, I have no problem with moving the body of those functions to their callers, and will do soon.

Reply:
Done in https://github.com/wxWidgets/wxWidgets/pull/25336/commits/95daea623880c1ba841c18e62f7f6b507b1ae832